DANS-KNAW/easy-convert-bag-to-deposit
# DD-450: copy some metadata files to data/migration

Fixes DD-450: copy some metadata files to data/migration

When applied it will
--------------------
* create copies of metadata files in `data/easy-migration`
* update the (tag)manifests only for changed/new/copied files
* 

Where should the reviewer @DANS-KNAW/easy nad @DANS-KNAW/dataversedans start?
------------------------------------------------

How should this be manually tested?
-----------------------------------

* run unit test AppSpec
* it should create two bags in `target/test/AppSpec/ingest-dir`
* a [breakpoint](https://github.com/DANS-KNAW/bagit-java/blob/e1a8e301b0e13521c3a76879f2807c8cc0086fae/src/main/java/gov/loc/repository/bagit/creator/AbstractCreateManifestsVistor.java#L55) shows calculations are only done for changed/copied/new files in the bag
* eas-validate-dans-bag (deploy when running on deasy) should only complain about empty emd.xml files
* it complains also about users on the original bags in `src/test/resources/bags/01`
* these validations imply that manifests are calculated correctly


Related pull requests on github
-------------------------------

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
